Chat Reporting Helper/assets/fo/lang/ru_ru.json

```
@@ -0,0 +1,30 @@
+{
+    "gui.socialInteractions.tooltip.report": "Пожаловаться на игрока в Mojang",
+    "gui.socialInteractions.narration.report": "Пожаловаться на %s в Mojang",
+    "gui.socialInteractions.tooltip.report.disabled": "Жалобы в Mojang в данный момент не доступны.",
+    "gui.socialInteractions.tooltip.report.no_messages": "На игрока %s можно подать жалобу в Mojang, но он ещё ничего не писал.",
+    "gui.socialInteractions.tooltip.report.not_reportable": "Этот игрок или сервер отключили жалобы в Mojang.",
+    "gui.chatReport.describe": "Дополнительная информация поможет Mojang сделать более обдуманное решение. Имейте в виду, что эти жалобы не винды админам сервера.",
```

```suggestion
    "gui.chatReport.describe": "Подробности помогут Mojang принять правильное решение. Имейте в виду, что эти жалобы не винды админам сервера.",
```
https://github.com/InventivetalentDev/minecraft-assets/blob/1.19.3/assets/minecraft/lang/ru_ru.json#L3302